src/applications/gi/containers/Modals.jsx

```diff
@@ -1011,24 +1063,50 @@ export class Modals extends React.Component {
         <div className="align-left">
           <h3>Which GI Bill benefit do you want to use?</h3>
         </div>
-        <p>
-          You may be eligible for multiple types of education and training
-          programs. Different programs offer different benefits, so it’s
-          important to choose the program that will best meet your needs. Use
-          this tool to compare programs and schools.
-        </p>
-        <p>
-          For detailed information on eligibility requirements and general
-          program benefits, visit{' '}
-          <a
-            href="/education/eligibility/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            this page
-          </a>
-          .
-        </p>
+        {environment.isProduction() ? (
+          <div>
+            {' '}
+            <p>
+              You may be eligible for multiple types of education and training
+              programs. Different programs offer different benefits, so it’s
+              important to choose the program that will best meet your needs.
+              Use this tool to compare programs and schools.
+            </p>
```

Comment:
Is this paragraph and the enclosing div needed for the PR, since only the next paragraph changes?